docs/reference_keymap_extras.md

```diff
@@ -34,6 +34,7 @@ These headers are located in [`quantum/keymap_extras/`](https://github.com/qmk/q
 |English (US International, Linux)|`keymap_us_international_linux.h`|                                    |
 |Estonian                         |`keymap_estonian.h`              |`sendstring_estonian.h`             |
 |Farsi                            |`keymap_farsi.h`                 |                                    |
+|EurKEY                           |`keymap_eurkey.h`                |                                    |
```

Comment:
```suggestion
|EurKEY                           |`keymap_eurkey.h`                |                                    |
|Farsi                            |`keymap_farsi.h`                 |                                    |
```

The farsi keymap was merged in the meantime, lets reorder that and then it is good for merge.